### PR TITLE
Show tox-tags install and fix usage

### PR DESCRIPTION
**Pull request by decentral1se**
Closes https://github.com/ansible/molecule/issues/1764.

#### PR Type

- Docs Pull Request

### CODE REVIEW

**Comment by decentral1se** [comment]
@themr0c 

Can I nag you for a review since I am trying to fix a ticket based on your recommendations :)

Thanks!

File: doc/source/testing.rst [hunk 1]
```diff
@@ -61,18 +56,18 @@ Run all functional tests.
...
 
-    The functional tests are a work in progress.  They need better structure
-    and reuse.
+    The functional tests are a work in progress. They need better structure and
+    reuse. They are also very slow and costly in terms of system resources.
 
 .. code-block:: bash
 
-    $ tox -t functional
+    $ tox -e functional
```
[the remark this answers is not in the document]
Comment (decentral1se) [reply doc/source/testing.rst]: It's a tag, see https://github.com/ansible/molecule/blob/master/tox.ini#L143.

You already do this? We documented it together in line 194 of https://etherpad.openstack.org/p/ansible-molecule?

Whatever about the `tox` CLI not showing tags correctly, which seems like an issue ... this docs change just makes it transparent for other contributors. Which is an improvement for now.

**Comment by themr0c** [comment]
The original issue was: https://github.com/ansible/molecule/issues/1764
Sorry i didn't work on it before.

**Comment by themr0c** [comment]
I am a little bit puzzled here because on this PR, on one hand we make sure we install tox-tags with the test-requirements.txt (which has been packaged, great job), and on the other hand we stop using it by replacing tox -t with tox -e.

**Comment by decentral1se** [comment]
OK, further confusion. After a quick chat on IRC and seeing https://github.com/AndreLouisCaron/tox-tags/issues/1#issuecomment-473938548, I will try to see about removing `tox-tags` altogether. `tox -l` does not list these tags ... but `tox -e unit` targets the tags ... it's weird ... anyway, closing for now!

**Comment by themr0c** [comment]
Seems that tox-tags is not working anymore, we should:

- remove tox-tags from test-requirements.txt
- remove all "tags" references from tox.ini
- in the documentation replace every `tox -t something` by `tox -e $(tox -l | grep lint | paste -d, -s -)`

And then for some better alternative to have something looking simpler in the doc

File: doc/source/testing.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (webknjaz) [reply doc/source/testing.rst]: @ssbarnea no, it's not an env, it's a factor (see tox docs). You **can** use it like this.

File: doc/source/testing.rst [hunk 2]
```diff
@@ -12,17 +12,12 @@ Dependencies
...
 .. code-block:: bash
 
     $ pip install tox
-
-Install the remaining requirements in a venv (optional).
-
-.. code-block:: bash
-
-    $ pip install -r test-requirements.txt -r requirements.txt
+    $ pip install -r test-requirements.txt
```
Comment (webknjaz) [inline doc/source/testing.rst]: Actually, users shouldn't have to install any requirements other than `tox` (and maybe its plugins).

**Comment by webknjaz** [comment]
@decentral1se the PR is mostly fine. No reason to discard it I think.

**Comment by decentral1se** [comment]
hehe, I think we all need to read the Tox documentation :)

Hoping @themr0c kills it in their PR!

**Comment by webknjaz** [comment]
@decentral1se apparently I confused things with some other tool a bit :)

**Pull request closed without merging**